### PR TITLE
Tx it

### PR DESCRIPTION
**Pull request by Salem-Luos**

### CODE REVIEW

File: src/pages/low/hardware_topics/electronic-design.md [hunk 1]
```diff
@@ -22,3 +23,12 @@ Luos' RS485 official connector is: <a href="https://octopart.com/df11-8dp-2ds%28
 Luos manages any type of microcontrollers, but they need to be added manually to the library. If your microcontroller is not supported yet, please contact us:
  - by mail: <a href="mailto:hello@luos.io">hello@luos.io</a>
  - on <a href="https://github.com/Luos-io/Luos/issues/new?assignees=nicolas-rabault&labels=porting&template=porting-request.md&title=%5BMCU+PORTING%5D+" target="_blank">GitHub</a>
+
+## PTP
```
Comment (Simonbdy) [inline src/pages/low/hardware_topics/electronic-design.md:27]: A short sentence of introduction of what is PTP and what it does is missing

File: src/pages/low/hardware_topics/minimal-requirements.md [hunk 2]
```diff
@@ -3,18 +3,18 @@ Luos Library uses hardware and software MCU ressources. Here are the minimum req
...
 ## Network consideration
 | MCU frequency | Bus frequency |
-| :---: | :---: | 
+| :---: | :---: |
 | >=48Mhz | 1Mbs |
 | >=32Mhz | 500Kbs |
-| >=16Mhz | 250Kbs | 
-| >=8Mhz | 100Kbs | 
+| >=16Mhz | 250Kbs |
+| >=8Mhz | 100Kbs |
```
Comment (Simonbdy) [inline src/pages/low/hardware_topics/minimal-requirements.md:20]: Mbs / Kbs is not clear. Does b stands for bit or Byte? Does s stands for /s?

**Review by Simonbdy — CHANGES_REQUESTED** [review]
I fixed a few typos, but I wrote 2 comments that need to be resolved:
 - A short introduction to / description of PTP
 - Fix the units (is kbs kB/s?)
 - Fix the conflict on the branch

File: src/pages/low/hardware_topics/electronic-design.md (hunk 1 above)
Comment (Salem-Luos) [reply src/pages/low/hardware_topics/electronic-design.md:27]: done

File: src/pages/low/hardware_topics/minimal-requirements.md (hunk 2 above)
Comment (Salem-Luos) [reply src/pages/low/hardware_topics/minimal-requirements.md]: already fix in a previous commit

**Review by Simonbdy — APPROVED** [review]

**Comment by jlangbridge** [comment]
Graphics file star_mounting.png isn't clearly readable for color deficient people (I can't read the yellow/grean/whatever part of the image)

**Review by jlangbridge — COMMENTED** [review]
All OK except for the graphics file

**Comment by Simonbdy** [comment]
@jlangbridge was right, I didn't even see `PTP C` in the image. The problem is not that it's green or yellow, it's that it's a very clear color with a very clear background. I suggest to put a darker color, like a darker green.

**Pull request merged**